src/sync/index.js

```diff
@@ -351,7 +116,7 @@ async function sync(db){
       console.log('Synced Topics & Oracles');
       // sync first and then update in case update object in current batch
       Promise.all(updatePromises).then(() =>{
-        console.log('Updated OracleResult & FinalResult');
+        console.log('Updated OracleResult & FinalResult\n');
```

Comment:
console.log should be auto newline, isnt it?

Reply:
I added this to "batch" the logs together and make it more readable:
```
searchlog TopicCreated
71053 - 71073: Retrieved 0 entries from TopicCreated
searchlog OracleResultSet
71053 - 71073: Retrieved 0 entries from OracleResultSet
searchlog DecentralizedOracleCreated
71053 - 71073: Retrieved 0 entries from DecentralizedOracleCreated
searchlog OracleResultVoted
71053 - 71073: Retrieved 0 entries from OracleResultVoted
searchlog CentralizedOracleCreated
71053 - 71073: Retrieved 0 entries from CentralizedOracleCreated
Synced Topics & Oracles
searchlog FinalResultSet
71053 - 71073: Retrieved 0 entries from FinalResultSet
Updated OracleResult & FinalResult

searchlog TopicCreated
71053 - 71073: Retrieved 0 entries from TopicCreated
searchlog OracleResultSet
71053 - 71073: Retrieved 0 entries from OracleResultSet
searchlog DecentralizedOracleCreated
71053 - 71073: Retrieved 0 entries from DecentralizedOracleCreated
searchlog OracleResultVoted
71053 - 71073: Retrieved 0 entries from OracleResultVoted
searchlog CentralizedOracleCreated
71053 - 71073: Retrieved 0 entries from CentralizedOracleCreated
Synced Topics & Oracles
searchlog FinalResultSet
71053 - 71073: Retrieved 0 entries from FinalResultSet
Updated OracleResult & FinalResult
```